beacon_node/beacon_chain/src/block_verification_types.rs

```diff
@@ -193,34 +170,60 @@ impl<E: EthSpec> RpcBlock<E> {
         Self::new(Some(block_root), block, blobs)
     }
 
+    /// Create an RpcBlock from an untrusted block, without checking that the blobs or columns are
```

Comment:
I think the docs for RpcBlock and RpcBlockInner need to be updated to reflect the fact that they could be constructed without checking for consistency